typescript/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@keep-network/tbtc-v2-ts",
-  "version": "0.1.0",
+  "name": "@keep-network/tbtc.ts",
+  "version": "2.0.0-dev",
```

Comment:
Let's not block on it but consider it as something that we still need to decide: I commented about it [in the issue](https://github.com/keep-network/tbtc-v2/issues/178#issuecomment-1096264677) and I think the package name should be `tbtc-v2` and the version should be `1.0.0-dev`.